src/client/context/channelContext.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const ChannelContext = React.createContext({
+  channel: null, // default value
+});
```

Comment:
What is this for?

Reply:
I thought you were talking about to create a context to the channel but now I got it, it should be UserContext.consumer inside the channel.